ros2action/ros2action/verb/send_goal.py

```diff
@@ -42,7 +42,8 @@ def add_arguments(self, parser, cli_name):
         arg = parser.add_argument(
             'goal',
             help="Goal request values in YAML format (e.g. '{order: 10}')")
-        arg.completer = ActionGoalPrototypeCompleter(action_type_key='action_type')
+        arg.completer = ActionGoalPrototypeCompleter(
+            action_type_key='action_type')
```

Comment:
It would be good to remove these formatting changes to keep the diff to be only the relevant change (replacing `spin_until_future_complete`.

Reply:
yes, formatting changes was my mistake, turns out I had some plugin in IDE (vs code) which applied the formatting implicitly while saving the file. i will restore it later today.

Reply:
Formatting changes are restored